### PR TITLE
Completed the color picker functionality on the PrintCardContainer component. 

### PR DESCRIPTION
**Pull request by Patrickcpoole**
Using the react-color library we now have the ability to choose which color we want the print card to print.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 node_modules
+<<<<<<< HEAD
```
Comment (ethoreson) [inline .gitignore]: @Patrickcpoole Please remove any merge conflict comments, otherwise app will crash

Comment (ethoreson) [reply .gitignore]: & when this is updated, all the build/... files will disappear from this PR & clean it up a lot.

File: src/components/Queue/components/PrintCardContainer.css [hunk 2]
```diff
@@ -1,31 +1,40 @@
 .printCardContainer {
```
Comment (ethoreson) [inline src/components/Queue/components/PrintCardContainer.css]: Changes requested:
display: flex --> display: inline-flex;
Add: grid-template-columns: 20% 20% 20% 20% 20%;
Delete: justify-content & align-items
Add: padding-left: 5em;

File: src/components/Queue/components/PrintCardContainer.jsx [hunk 3]
```diff
@@ -1,32 +1,70 @@
...
+const PrintCardContainer = ({ data }) => {
+	const [circleColor, setcircleColor] = useState(data.color);
+	const [hoverState, setHoverState] = useState(false);
+
+	const colorCircleStyle = {
+		backgroundColor: `${circleColor}`
+	};
+
+	const handleChangeComplete = color => {
+		console.log('New color will be', color);
```
Comment (ethoreson) [inline src/components/Queue/components/PrintCardContainer.jsx]: Please delete console.logs, when they pile up it makes the console hard to read

File: src/components/Queue/components/PrintCardContainer.css (hunk 2 above)
Comment (Patrickcpoole) [reply src/components/Queue/components/PrintCardContainer.css]: Changed!

File: src/components/Queue/components/PrintCardContainer.jsx (hunk 3 above)
Comment (Patrickcpoole) [reply src/components/Queue/components/PrintCardContainer.jsx]: Deleted the console.logs

File: .gitignore (hunk 1 above)
Comment (Patrickcpoole) [reply .gitignore]: Conflicts resolved.

**Pull request merged**